charts/cp-control-center/values.yaml

```diff
@@ -78,8 +78,8 @@ prometheus:
   ## ref: https://github.com/prometheus/jmx_exporter
   jmx:
     enabled: true
-    image: solsson/kafka-prometheus-jmx-exporter@sha256
-    imageTag: 6f82e2b0464f50da8104acd7363fb9b995001ddff77d248379f8788e78946143
+    image: eu.gcr.io/bbc-registry/kafka-prometheus-jmx-exporter
+    imageTag: 32d09291efb2b25aee6b5f694506196fed5f1365946af5884ce86a0e2039915e
```

Comment:
Using the hash is not our best-practice.
You should modify the Dockerfile (cf: https://github.com/blablacar/dockerfiles/pull/2391#pullrequestreview-2148352845) to have a version

Reply:
The thing here is that there is not tag on the upstream version so I don't like to add semver for something we don't own niethr change anything expect doing a simple copy in our registry ?